lib/fielded_search.rb

```diff
@@ -0,0 +1,63 @@
+class FieldedSearch
+  attr_reader :field_pairs
+
+  def initialize(field_pairs)
+    @field_pairs = field_pairs
+  end
+
+  def present?
+    return false unless field_pairs
+    to_s.present?
+  end
+
+  def value_for(key)
+    if field_pairs && field_pairs.key?(key)
+      field_pairs[key]
+    end
+  end
+
+  def to_s
```

Comment:
ah yes, my bad. I guess that is part of the reason I find these confusing -- It feels like we are monkeypatching even though we aren't. 

like in `organization.rb` where we defined `self.all` and `self.find` which are Rails methods. That was just a Ruby class, but defining the same methods at Rails but defining them differently proved to be quite confusing when refactoring.  (https://github.com/18F/C2/pull/870/files) 

What do you see as the downside as giving these methods more specific / descriptive names?